lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -84,6 +88,9 @@ export default function NarrativeItinerariesHeader({
               <FormattedMessage id="components.NarrativeItinerariesHeader.viewAll" />
             </IconWithText>
           </button>
+          <h1 style={{ display: 'none' }}>
```

Comment:
For use with screen readers only, use CSS `visibility: hidden` and set a zero-width or zero-height (see other comment regarding `display: none`).

Reply:
Sorry, don't render this `<h1>` at all. It was not going to be shown to screen readers (see https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#accessibility_concerns). Rather, set a zero-size to the `<h1>` above this one as needed.

Reply:
Thank you so much for catching this!! I think removing this h1 leaves the itinerary narrative without a header, but I can fix both with `height: 0; width: 0; overflow: hidden;`